tutorials/backend/hasura/tutorial-site/content/custom-business-logic/2-remote-schemas.md

```diff
@@ -25,60 +25,68 @@ const fetch = require('node-fetch');
 
 const typeDefs = gql`
   type auth0_profile {
-      email: String
-      picture: String
-    }
+    email: String
+    picture: String
+  }
 
-    type Query {
-      auth0: auth0_profile
-    }
+  type Query {
+    auth0: auth0_profile
+  }
 `;
 
-function getProfileInfo(user_id){
-    const headers = {'Authorization': 'Bearer '+process.env.AUTH0_MANAGEMENT_API_TOKEN};
-    console.log(headers);
-    return fetch('https://' + process.env.AUTH0_DOMAIN + '/api/v2/users/'+user_id,{ headers: headers})
-        .then(response => response.json())
-}
+function getProfileInfo(user_id) {
+  const headers = {
+    Authorization: `Bearer ${process.env.AUTH0_MANAGEMENT_API_TOKEN}`,
+  };
+  console.log(headers);
 
+  return fetch(`https://${process.env.AUTH0_DOMAIN}/api/v2/users/${user_id}`, {
+    headers,
+  });
+}
 
 const resolvers = {
-    Query: {
-        auth0: (parent, args, context) => {
-          // read the authorization header sent from the client
-          const authHeaders = context.headers.authorization || '';
-          const token = authHeaders.replace('Bearer ', '');
-          // decode the token to find the user_id
-          try {
-            if (!token) {
-              return 'Authorization token is missing!';
-            }
-            const decoded = jwt.decode(token);
-            const user_id = decoded.sub;
-            // make a rest api call to auth0
-            return getProfileInfo(user_id).then( function(resp) {
-              console.log(resp);
-              if (!resp) {
-                return null;
-              }
-              return {email: resp.email, picture: resp.picture};
-            });
-          } catch(e) {
-            console.log(e);
-            return null;
-          }
+  Query: {
+    auth0: (parent, args, context) => {
+      // read the authorization header sent from the client
+      const authHeaders = context.headers.authorization || '';
+      const token = authHeaders.replce('Bearer ', '');
```

Comment:
This looks like a typo.

Reply:
Fixed one bug and added another, classic 😄